instructor-notes/08_pull-requests-basics.md

```diff
@@ -35,29 +35,47 @@ In the most recent session, participants will have created two branches:
 
 * Checkout the `add-renv` branch locally.
 * We will make a small change to `download-fastq.sh` to skip downloading the `fastq` files if they already exist.
-    * Check if files exist using the `-e` flag as outlined in [`solutions/fileexists_download-fastq.sh`](solutions/fileexists_download-fastq.sh).
-    This sets us up to have a merge conflict with the `add-fastp` branch in the next session.
+  * Check if files exist using the `-e` flag as outlined in [`solutions/fileexists_download-fastq.sh`](solutions/fileexists_download-fastq.sh).
+  This sets us up to have a merge conflict with the `add-fastp` branch in the next session.
 * Commit the change to the `add-renv` branch.
 * Push the changes to the remote `add-renv` branch.
 * Instruct participants to navigate to their remote repositories in the browser to see their new commit pushed.
   * Point out the banner that says changes were recently pushed to the `add-renv` branch.
 
+### Protecting `main`
+
+* Navigate to the settings of the repository, where you will demonstrate how to protect the `main` branch.
+* In the left sidebar, click on "Branches".
+* Under "Branch protection rules", click on "Add branch ruleset".
+  * Stress that protecting the `main` branch when working in their own repositories is not practical, and that participants should not follow along in setting this up in their fork.
+  The reason why: If you want to keep your `main` branch up-to-date with the upstream repository, you will need to merge changes from the upstream `main` branch into your own `main` branch.
+  This is painful to do if your `main` branch is protected.
```

Comment:
THANK YOU FOR ADDING THIS! This is exactly why I was back looking at the rules and thinking about it. The feedback is terrible: syncing upstream simply fails with no explanation. I clicked it three times before realizing what was happening.